src/inmanta/data/__init__.py

```diff
@@ -2644,26 +2644,35 @@ async def get_latest_version(cls, environment: uuid.UUID, resource_id: m.Resourc
             return resources[0]
 
     @classmethod
-    def _get_released_resources_partial_base_query(cls):
+    def _get_released_resources_base_query(
+        cls, select_clause: str, environment_filter: Tuple[QueryType, object], offset: int
```

Comment:
if query type on environment_filter is not '=' this query will return nonsense? 

This signature makes it seem that it supports any query type.